### PR TITLE
0.7.0

### PR DESCRIPTION
**Pull request by JohnSzingerCR**

### CODE REVIEW

File: drp_aa_mvp/data_rights_request/views.py [hunk 1]
```diff
@@ -376,12 +549,12 @@ def create_drp_request_transaction(user_identity, covered_biz, request_json, res
         audience                = request_json.aud,
         expires_time            = request_json.exp,
         issued_time             = request_json.iat,
-        name                    = user_identity.first_name,         #user_identityfull_name,
+        name                    = user_identity.user_identity.full_name,
```
Comment (rrix) [inline drp_aa_mvp/data_rights_request/views.py]: why is user_identity duplicated here?

File: drp_aa_mvp/data_rights_request/views.py [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
+
+
+
+#root_utl = os.environ['REQUEST_URI']
+#print (f"****  root_url = {root_utl}")
 
+auth_agent_drp_id       = 'CR_AA_DRP_ID_001'
+auth_agent_callback_url = "http://127.0.0.1:8001/update_status" #f"{os.environ.get('SERVER_NAME')}/update_status" 
+
+signing_key = None
```
Comment (rrix) [inline drp_aa_mvp/data_rights_request/views.py]: why not just call generate_key() here?

File: drp_aa_mvp/data_rights_request/views.py (hunk 1 above)
Comment (rrix) [reply drp_aa_mvp/data_rights_request/views.py]: actually where is full_name defined at all?

**Review by rrix — APPROVED** [review]
good stuff :)

File: drp_aa_mvp/data_rights_request/views.py (hunk 1 above)
Comment (JohnSzingerCR) [reply drp_aa_mvp/data_rights_request/views.py]: typo - will fix

File: drp_aa_mvp/data_rights_request/views.py (hunk 2 above)
Comment (JohnSzingerCR) [reply drp_aa_mvp/data_rights_request/views.py]: hmm, yeah good point

**Comment by rrix** [comment]
ah fuddge

**Pull request merged**